deploy/028_deploy_linea_adapter.ts

```diff
@@ -14,7 +14,9 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       WETH[chainId],
       L1_ADDRESS_MAP[chainId].lineaMessageService,
       L1_ADDRESS_MAP[chainId].lineaTokenBridge,
-      L1_ADDRESS_MAP[chainId].lineaUsdcBridge,
+      // TODO: Requires USDC Linea address to be added, currently only USDC.e is set for Linea
+      USDC[chainId],
```

Comment:
Have you tested this via an actual deployment? I think it's dependent on us adding Linea USDC definition to the constants repo.

A short-term workaround would be to import `USDCe` instead; this is OK in this context because the Linea address will be unchanged through the migration.

Reply:
No that's why I added a TODO here. this won't work, but I also don't anticipate us using this deploy script until after we've updated const. For now I've updated to USDCe

Reply:
f908e3888dd80f1f7c8621966edcbd5de9aae7fe